e2e-tests/manifests/rapidast-vapi-configmap-urls-exclusions.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: v1
+data:
+  config.yaml: |+
+    config:
+      configVersion: 5
+
+    # `application` contains data related to the application, not to the scans.
+    application:
+      shortName: "v5-none-release-test"
+      url: "http://vapi:5000"
+
+    scanners:
+      zap:
+      # define a scan through the ZAP scanner
+        apiScan:
+          apis:
+            apiUrl: "http://vapi:5000/docs/openapi.json"
+
+        passiveScan:
+          # optional list of passive rules to disable
+          disabledRules: "2,10015,10027,10096,10024,10098,10023,10105"
+
+        activeScan:
+          policy: API-scan-minimal
+
+        container:
```

Comment:
some of these config fields are maybe unnecessary for this test, e.g. this `container` section and perhaps some of `miscOptions`

Reply:
The main reason I used the same config file was to make sure the alerts match the ones from the test that checks for exclusions when there are no rules. I just wanted to avoid any mismatch between the two tests that could cause some alerts to be missing in the second one (where exclusions are tested). That said, you're right, the options you mentioned shouldn’t affect the results, so I've updated the config file.